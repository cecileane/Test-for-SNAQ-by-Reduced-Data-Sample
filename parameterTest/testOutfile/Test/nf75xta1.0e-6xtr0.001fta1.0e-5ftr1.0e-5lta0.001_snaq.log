optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-5,
                       xtolAbs=1.0e-6, xtolRel=0.001.
 max number of failed proposals = 75, liktolAbs = 0.001.
rootname for files: nf75xta1.0e-6xtr0.001fta1.0e-5ftr1.0e-5lta0.001_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 41779
seed: 41779 for run 1
Sat Oct  1 16:59:31 2016

 BEGIN SNaQ for run 1, seed 41779 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 161 (failures: 75) with -loglik=165.41478 and ht_min=[0.33546,0.86609,0.46105,0.03807,10.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 53		 53		 4	 |	 0		 0		 --
mvtarget	 37		 37		 0	 |	 0		 0		 --
chdir		 13		 13		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 56		 31		 1	 |	 16		 16		 --
Total		 161		 135		 6	 |	 16		 16		 6
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.4

 FINISHED SNaQ for run 1, -loglik of best 165.4147783791799
(2,1,((4,(6,(5,#H9:::0.3354608483030789):10.0):0.8660902397745603):0.03806519960055376,(3)#H9:::0.6645391516969211):0.4610518044054217);
---------------------
seed: 65019 for run 2
Sat Oct  1 16:59:40 2016

 BEGIN SNaQ for run 2, seed 65019 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 221 (failures: 75) with -loglik=155.49626 and ht_min=[0.12573,0.33535,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 74		 74		 4	 |	 0		 0		 --
mvtarget	 59		 59		 1	 |	 0		 0		 --
chdir		 24		 24		 3	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 62		 45		 1	 |	 2		 2		 --
Total		 221		 204		 10	 |	 3		 3		 3
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 155.49625734561005
((((5,6):0.33534700991566996,3):0.12573373019571793,(1,(2)#H7:::0.6269753030775719):2.155859354452936):2.155859354452936,4,#H7:::0.37302469692242807);
---------------------

Sat Oct  1 16:59:54 2016
MaxNet is ((((5,6):0.33534700991566996,3):0.12573373019571793,(1,(2)#H7:::0.6269753030775719):2.155859354452936):2.155859354452936,4,#H7:::0.37302469692242807); 
with -loglik 155.49625734561005
