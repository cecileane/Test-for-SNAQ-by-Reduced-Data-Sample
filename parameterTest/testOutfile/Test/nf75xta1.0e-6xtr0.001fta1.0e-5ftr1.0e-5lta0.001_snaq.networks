((((5,6):0.33534700991566996,3):0.12573373019571793,(1,(2)#H7:::0.6269753030775719):2.155859354452936):2.155859354452936,4,#H7:::0.37302469692242807);, with -loglik 155.49625734561005 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(((5,6):0.32597989833592717,3):0.0,(1,(#H7:::0.3187721860934466,2):6.23231876536364):1.1030086623053592,(4)#H7:::0.6812278139065534);, with -loglik 158.662980367558
((((5,6):0.32330575427945735,3):0.0,#H7:::0.4826010555938788):0.5758437400750246,4,((1)#H7:::0.5173989444061212,2):6.497022431893181);, with -loglik 170.71611601371205
((((5,6):0.3912303437310873,3):0.0)#H7:1.0294038920377504::0.7508419313063527,4,((1,#H7:2.325920275481692::0.24915806869364732):10.0,2):1.5872086381352482);, with -loglik 173.38886413946312
