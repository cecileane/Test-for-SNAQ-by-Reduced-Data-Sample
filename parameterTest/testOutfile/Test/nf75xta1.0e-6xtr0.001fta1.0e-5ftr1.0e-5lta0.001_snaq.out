((((5,6):0.33534700991566996,3):0.12573373019571793,(1,(2)#H7:::0.6269753030775719):2.155859354452936):2.155859354452936,4,#H7:::0.37302469692242807); -Ploglik = 155.49625734561005
 Dendroscope: ((((5,6):0.33534700991566996,3):0.12573373019571793,(1,(2)#H7):2.155859354452936):2.155859354452936,4,#H7);
 Elapsed time: 22.180375396 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((((5,6):0.33534700991566996,3):0.12573373019571793,(1,(2)#H7:::0.6269753030775719):2.155859354452936):2.155859354452936,4,#H7:::0.37302469692242807);, with -loglik 155.49625734561005
 (2,1,((4,(6,(5,#H9:::0.3354608483030789):10.0):0.8660902397745603):0.03806519960055376,(3)#H9:::0.6645391516969211):0.4610518044054217);, with -loglik 165.4147783791799
-------
